x-pack/test/functional/page_objects/space_selector_page.ts

```diff
@@ -120,6 +120,22 @@ export class SpaceSelectorPageObject extends FtrService {
     await this.testSubjects.setValue('euiColorPickerAnchor', hexValue);
   }
 
+  async openSolutionViewSelect() {
+    const solutionViewSelect = await this.testSubjects.find('solutionViewSelect');
+    const classes = await solutionViewSelect.getAttribute('class');
+
+    const isOpen = classes?.includes('isOpen') ?? false;
+    if (!isOpen) {
+      await solutionViewSelect.click();
+    }
+  }
+
+  async changeSolutionView(solution: 'es' | 'oblt' | 'security' | 'classic') {
```

Comment:
nit: may we use `Space[solution]` type here?

Reply:
As discussed offline, let's swap the type in future PR to avoid a full CI cycle just for that. 👍